rdt-open-reader/src/main/java/io/ona/rdt/application/RDTApplication.java

```diff
@@ -72,6 +73,11 @@ public void onCreate() {
         if (sharedPreferences.getPreference(IS_IMG_SYNC_ENABLED).isEmpty()) {
             sharedPreferences.savePreference(IS_IMG_SYNC_ENABLED, String.valueOf(true));
         }
+
+        // set locale for MLS
+        com.vijay.jsonwizard.utils.AllSharedPreferences allSharedPreferences
```

Comment:
is there a reason we're not importing this?

Reply:
We have two types of `AllSharedPreferences`, this particular one is from native forms, not core.

Reply:
Does that mean we can not add an `import com.vijay.jsonwizard.utils.AllSharedPreferences` at the top of the file?